stt_engines/deepspeech/main.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+stt_deepspeech_install () {
+  set -e # Don't attempt to continue if an error occurs
+  jv_install python3 python3-pip virtualenv
+  cd "$(dirname "${BASH_SOURCE[0]}")"
+  virtualenv -p python3 .
+  source bin/activate
+  pip3 install deepspeech
+  deactivate
+  wget https://github.com/mozilla/DeepSpeech/releases/download/v0.7.0/deepspeech-0.7.0-models.pbmm
+  wget https://github.com/mozilla/DeepSpeech/releases/download/v0.7.0/deepspeech-0.7.0-models.scorer
+  cd "$jv_dir"
+}
+
+# Check if the needed files exist and install DeepSpeech if they don't
+[[ ( -f "$(dirname "${BASH_SOURCE[0]}")/deepspeech-0.7.0-models.pbmm" ) && \
+( -f "$(dirname "${BASH_SOURCE[0]}")/deepspeech-0.7.0-models.pbmm" ) && \
+( -d "$(dirname "${BASH_SOURCE[0]}")/bin" ) && \
+( -f "$(dirname "${BASH_SOURCE[0]}")/bin/deepspeech" ) ]] || {
```

Comment:
Check if the DeepSpeech model, scorer and binary (and the bin directory of the virtualenv) exist